tests/hcaptcha.spec.js

```diff
@@ -127,4 +127,117 @@ describe('hCaptcha', () => {
+            const ExpectHost = "https://test.com";
+
+            instance = ReactTestUtils.renderIntoDocument(<HCaptcha
+                    apihost={ExpectHost}
+                    sitekey={TEST_PROPS.sitekey}
+                />);
+
+            const script = document.querySelector("head > script");
+            expect(script.src).toEqual(expect.stringContaining(ExpectHost));
+            expect(script.src).not.toEqual(expect.stringContaining(`apihost=${encodeURIComponent(ExpectHost)}`));
+        });
+
+        it("assethost should be found when prop is set", () => {
+            const ExpectHost = "https://test.com";
+
+            instance = ReactTestUtils.renderIntoDocument(<HCaptcha
+                    assethost={ExpectHost}
+                    sitekey={TEST_PROPS.sitekey}
+                />);
+
+            const script = document.querySelector("head > script");
+            expect(script.src).toEqual(expect.stringContaining(`assethost=${encodeURIComponent(ExpectHost)}`));
+        });
+
+        it("endpoint should be found when prop is set", () => {
+            const ExpectHost = "https://test.com";
+
+            instance = ReactTestUtils.renderIntoDocument(<HCaptcha
+                    endpoint={ExpectHost}
+                    sitekey={TEST_PROPS.sitekey}
+                />);
+
+            const script = document.querySelector("head > script");
+            expect(script.src).toEqual(expect.stringContaining(`endpoint=${encodeURIComponent(ExpectHost)}`));
+        });
+
+        it("imghost should be found when prop is set", () => {
+            const ExpectHost = "https://test.com";
+
+            instance = ReactTestUtils.renderIntoDocument(<HCaptcha
+                    imghost={ExpectHost}
+                    sitekey={TEST_PROPS.sitekey}
+                />);
+
+            const script = document.querySelector("head > script");
+            expect(script.src).toEqual(expect.stringContaining(`imghost=${encodeURIComponent(ExpectHost)}`));
+        });
+
+        it("reportapi should be found when prop is set", () => {
+            const ExpectHost = "https://test.com";
+
+            instance = ReactTestUtils.renderIntoDocument(<HCaptcha
+                    reportapi={ExpectHost}
+                    sitekey={TEST_PROPS.sitekey}
+                />);
+
+            const script = document.querySelector("head > script");
+            expect(script.src).toEqual(expect.stringContaining(`reportapi=${encodeURIComponent(ExpectHost)}`));
+        });
+
+        it("hl should be found when prop languageOverride is set", () => {
+            instance = ReactTestUtils.renderIntoDocument(<HCaptcha
+                    languageOverride="fr"
+                    sitekey={TEST_PROPS.sitekey}
+                />);
+
+            const script = document.querySelector("head > script");
+            expect(script.src).toEqual(expect.stringContaining("hl=fr"));
```

Comment:
Any reasons not to use the shorter version `expect(script.src).toContain("hl=fr");` ?

Reply:
Whats the shorter version?

Reply:
Instead of:
`expect(script.src).toEqual(expect.stringContaining("hl=fr"));`
it can be:
`expect(script.src).toContain("hl=fr");`

Reply:
Honestly forgot that was an option, updated!